examples/data-objects/codemirror/src/codemirror.ts

```diff
@@ -273,16 +276,11 @@ class SmdeFactory implements IFluidDataStoreFactory {
         dataTypes.set(mapFactory.type, mapFactory);
         dataTypes.set(sequenceFactory.type, sequenceFactory);
 
-        const runtime = FluidDataStoreRuntime.load(
+        return createDataStoreRuntimeWithRouter(
```

Comment:
Is codemirror being used as the example here so you don't touch (Pure)DataObject and everything that extends it?  (Pure)DataObject doesn't have the `load(runtime, context)` method anymore, and I think it would be better to conform the API to that more than some other example component. (Unless the plan is to move (Pure)DataObject in this direction as well).

Is there merit to splitting out the datastore creation part here?  As in createDataStoreRuntimeWithRouter would add the mixin but then the routerCtor(runtime, context) part happens outside?  Also asking because while it's clear enough what's happening here, the method name feels a bit off to me (maybe just me though, since I can't really think of a more descriptive alternative)

Reply:
CodeMirror (as well as all other examples that are using registerRequestHandler are not based off Aqueduct. Aqueduct already uses mixin model, so it's take care for all of DataObjects.
While on one hand I'd favor moving all these examples to be based off Aqueduct, we also need to have examples (and feel the pain) of not using it.

As for your second observation, @urtisman had exactly same observation! So I'm changing it in latest iteration to split creation of factory and usage of factory. Please take a look.